scripts/install_ceres.sh

```diff
@@ -17,9 +17,12 @@
 set -o errexit
 set -o verbose
 
+VERSION="1.13.0"
+
 # Build and install Ceres.
 git clone https://ceres-solver.googlesource.com/ceres-solver
 cd ceres-solver
+git checkout tags/${VERSION}
```

Comment:
Clone directly the tag to save time if they are huge differences from master and tag?

Reply:
The only way to do this I know is to use `git clone --branch` which also accepts tags. But if there is a tag and a branch with the same name, this will give us the wrong thing, so I had to use this. 

https://stackoverflow.com/questions/791959/download-a-specific-tag-with-git

Reply:
Yes I was thinking to `git clone --branch` and indeed it does not work in this case.

Reply:
Using git fetch you can do something like this:
```
mkdir ceres-solver
cd ceres-solver
git init
git remote add origin https://ceres-solver.googlesource.com/ceres-solver
git fetch --depth 1 origin "tags/${VERSION}"
git checkout -b "${VERSION}" FETCH_HEAD
```

Reply:
@damienrg With the short history of Cartographer, this barely saves space. I think this is not worth the extra lines for now.